test/ReadsUtils_server_test.py

```diff
@@ -2779,7 +2809,7 @@ def test_upload_reads_from_web_direct_download_paired_ends(self):
         self.assertEqual(d['read_orientation_outward'], 0)
         self.assertEqual(d['insert_size_mean'], None)
         self.assertEqual(d['insert_size_std_dev'], None)
-        self.check_lib(d['lib1'], 1341276, file_name, '04dac97530faa081c966e79af7c90883')
+        self.check_lib(d['lib1'], 1341276, file_name, '1c58d7d59c656db39cedcb431376514b')
```

Comment:
@MrCreosote It was a bug that I found out. 

previously when I called _download_web_file for rev file, I didn't pass rev_file=True argument.